composer.json

```diff
@@ -12,33 +12,33 @@
             "type"    : "package",
             "package" : {
                 "name"    : "wet-boew/wet-boew",
-                "version" : "v4.0.28.1",
+                "version" : "v4.0.29",
```

Comment:
If we merge in all my pending merge requests to use https://github.com/drupalwxt/composer-extdeps then we won't need to update this across all projects.